engine/collection/message_hub/message_hub.go

```diff
@@ -249,7 +249,7 @@ func (h *MessageHub) processQueuedMessages(ctx context.Context) error {
 // processQueuedTimeout performs actual processing of model.TimeoutObject, as a result of successful invocation
 // broadcasts timeout object to consensus committee.
 // No errors are expected during normal operations.
-func (h *MessageHub) processQueuedTimeout(timeout *model.TimeoutObject) error {
+func (h *MessageHub) processQueuedTimeout(timeout *messages.ClusterTimeoutObject) error {
 	logContext := h.log.With().
```

Comment:
It would be great to include the tick number in the log, so we have visibility into when we're relying on re-broadcasts.

Reply:
yup, added it.